# Findings 
Here you find a summary of the initial findings

## Some stats

In [2]:
import pandas as pd
df = pd.read_csv('../data/archaeology_metadata.csv')
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 158254 entries, 0 to 158253
Data columns (total 13 columns):
 #   Column                                                Non-Null Count   Dtype  
---  ------                                                --------------   -----  
 0   dsPersistentId                                        158254 non-null  object 
 1   publicationStatus                                     158254 non-null  object 
 2   title                                                 158254 non-null  object 
 3   dsDescriptionValue                                    158254 non-null  object 
 4   dansSpatialPointX                                     56479 non-null   object 
 5   dansSpatialPointY                                     56479 non-null   object 
 6   dansSpatialPointScheme                                56304 non-null   object 
 7   dansSpatialBoxNorth                                   4432 non-null    object 
 8   dansSpatialBoxEast                          

In total, the archaeology data station contains 158254 published datasets. 
About 60000 have geospatial information in the metadata, either in the form of points (x, y), or a bounding box. 

--> About* 1/3 of the datasets have spatial metadata available. 

### Inspect the data with missing explicit geospatial metadata

In [4]:
df_nogeo = df[df.dansSpatialBoxNorth.isna() & df.dansSpatialPointX.isna()]

In [5]:
df_nogeo

dsPersistentId publicationStatus  \
109     doi:10.17026/dans-246-zkdb         Published   
128     doi:10.17026/dans-zaz-bggb         Published   
164     doi:10.17026/dans-xgk-zzky         Published   
167     doi:10.17026/dans-zaa-rsht         Published   
304     doi:10.17026/dans-x8b-vxz8         Published   
...                            ...               ...   
158210      doi:10.17026/AR/VEIR0Z         Published   
158216      doi:10.17026/AR/WUTNMU         Published   
158232      doi:10.17026/AR/XZ57NT         Published   
158233      doi:10.17026/AR/TAXDID         Published   
158234      doi:10.17026/AR/CHWIDU         Published   

                                                    title  \
109                           Rotterdam Noordsingel 90-92   
128     Verslag van een verkennend booronderzoek te Ge...   
164     Verslag van een verkennend booronderzoek op de...   
167                             Nesserlaan 2 te Witmarsum   
304     Plangebieden N670 Yerseke, N670 Kloetinge en N...   
...                                                   ...   
158210  Archeologische Begeleiding Wind A16 kabeltracé...   
158216  Oudemolen, Groot Boerenbos (Gemeente Tynaarlo,...   
158232  Archeologisch Bureauonderzoek. Burgemeester va...   
158233  Archeologisch Bureauonderzoek. Twee tracés per...   
158234  Archeologisch Bureauonderzoek. Villa Aerdenhou...   

                                       dsDescriptionValue dansSpatialPointX  \
109                                     onderzoeksrapport               NaN   
128                                     onderzoeksrapport               NaN   
164                                     onderzoeksrapport               NaN   
167     Aan het plangebied was een onbekend verwachtin...               NaN   
304     Zoals op basis van het bureauonderzoek werd ve...               NaN   
...                                                   ...               ...   
158210  Tussen oktober 2021 en juni 2022 is door Antea...               NaN   
158216  Het inventariserend booronderzoek is in twee f...               NaN   
158232  Archeologisch bureauonderzoek naar de aanwezig...               NaN   
158233  Archeologisch Bureauonderzoek naar de aanwezig...               NaN   
158234  Archeologisch bureauonderzoek naar de aanwezig...               NaN   

       dansSpatialPointY dansSpatialPointScheme dansSpatialBoxNorth  \
109                  NaN                    NaN                 NaN   
128                  NaN                    NaN                 NaN   
164                  NaN                    NaN                 NaN   
167                  NaN                    NaN                 NaN   
304                  NaN                    NaN                 NaN   
...                  ...                    ...                 ...   
158210               NaN                    NaN                 NaN   
158216               NaN                    NaN                 NaN   
158232               NaN                    NaN                 NaN   
158233               NaN                    NaN                 NaN   
158234               NaN                    NaN                 NaN   

       dansSpatialBoxEast dansSpatialBoxSouth dansSpatialBoxWest  \
109                   NaN                 NaN                NaN   
128                   NaN                 NaN                NaN   
164                   NaN                 NaN                NaN   
167                   NaN                 NaN                NaN   
304                   NaN                 NaN                NaN   
...                   ...                 ...                ...   
158210                NaN                 NaN                NaN   
158216                NaN                 NaN                NaN   
158232                NaN                 NaN                NaN   
158233                NaN                 NaN                NaN   
158234                NaN                 NaN                NaN

## Possibly useful variables 

| variable | description | example | source doi |
|---|---|---|---|
| dansTemporalSpatial:dansSpatialPoint | Has an X, Y and Scheme specification | 6.11557, 52.78667, longitude/latitude (degrees) |  10.17026/25AR/25AAV0BL | 
| dansTemporalSpatial:dansSpatialCoverageControlled |  | Netherlands | 10.17026/25AR/25AAV0BL | 
| dansTemporalSpatial:dansSpatialCoverageText | | 	"Sint-Michielsgestel", "NLD" | 10.17026/AR/KDIBWJ | 
| dansRelationMetadata:dansCollection:title | | Kalverstraat 11 Steenwijk | 10.17026/25AR/25AAV0BL | 
| ore:aggregates:schema:name | Contains the name of a file, often with location information | 115-261_Steenwijk_Kalverstraat 11.pdf / location.xml |  10.17026/25AR/25AAV0BL \ 10.17026/AR/KDIBWJ | 
| dansRelationMetadata:dansCollection:title | Title of the dataset, can contain clues on geospatial info |  Kalverstraat 11 Steenwijk | 10.17026/25AR/25AAV0BL | 
| citation:dsDescriptionValue | Contains a free description of the dataset | | | 
| ore:describes:language | Language of the report/dataset, might give a clue in toponym disambiguation step |  | | 




### Location files 
- This [location.xml](https://archaeology.datastations.nl/file.xhtml?fileId=3746676&version=1.0) is restricted